kong/pdk/service/request.lua

```diff
@@ -653,7 +654,7 @@ local function new(self)
         error("mime must be a string", 2)
       end
       if not mime then
-        mime = ngx.req.get_headers()[CONTENT_TYPE]
+        mime = get_header(CONTENT_TYPE)
```

Comment:
Here `get_header` guarantee `Content-Type` header's value is a string. `get_headers()[CONTENT_TYPE]` may returns a table if downstream sends multiple `Content-Type` headers.
From https://github.com/Kong/kong/pull/13981/files#diff-fab5190a31719b5022535ea42add0ef30683a4fe22ba208fd0976908e13c0d3bL664 we can see that `mime` here must be a string.